resource/reactivedrop_english.txt

```diff
@@ -1660,6 +1660,12 @@
 "nb_mission_title"	"Mission:"
 "nb_mission_options"	"MISSION OPTIONS"
 "nb_difficulty_title"	"Difficulty"
+"nb_speedruntime_title"	"Speedrun Time"
+// %d are the minutes, %02d are the seconds. examples - 1:42, 3:10, 1:07
```

Comment:
translator comment might be confusing because setting this to `%02d:%d` will show the numbers in the same order, just change which one is zero-padded

Reply:
should i just remove the comment

Reply:
up to you; could also change it to `the first %d is minutes and the second %d is seconds (printf formatting)`

There's another date format for the stats site that references printf and it hasn't caused any confusion